data/category-porn

```diff
@@ -11,17 +15,19 @@ include:illusion
 include:illusion-nonofficial
 include:jable
 include:javbus
+include:javcc
 include:javdb
-include:javfinder
 include:javwide
 include:johren
-include:livejasmin
 include:metart
+include:nutaku
 include:picacg
 include:pornhub
-include:scoreland
```

Comment:
The scoreland is belonging to The SCORE Group, so I changed this category name to make it include all the domains which are belonging that organization. The now name of this category is `thescoregroup`.